src/main/java/EtfRecommendService/user/User.java

```diff
@@ -85,4 +85,8 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hashCode(password);
     }
+
+    public void profileImgUpdate(String updateUrl) {
```

Comment:
언어 불문하고 보통 함수 이름은 동사+목적어 형태로 짓는 편이에요.
```suggestion
    public void updateProfileImg(String updateUrl) {
```

Reply:
네